app/assets/stylesheets/ui_components.scss.erb

```diff
@@ -7,3 +7,12 @@
 @import 'bootstrap-chosen';
 
 @import 'ui_components/*';
+
+<% UiComponents::Engine.components_paths.each do |path| %>
+  <%
```

Comment:
Right, we don't. The documentation is just confusing. http://www.rubydoc.info/github/sstephenson/sprockets/Sprockets%2FContext%3Arequire_asset

Then why do we even need to do this in the scss file? Can't we move it to `engine.rb`?